rover_msgs/Obstacle.lcm

```diff
@@ -3,4 +3,5 @@ package rover_msgs;
 struct Obstacle {
 	double bearing; // from straight ahead
 	double distance; // from straight ahead
+	double rightBearing;
```

Comment:
Could you move the rightBearing to be under the bearing?

Reply:
Fixed